### PR TITLE
Add guide for using vim as ruby ide

### PR DESCRIPTION
**Pull request by wsdjeg**
### PR Prelude

Thank you for working on SpaceVim! :)

Please complete these steps and check these boxes before filing your PR:

- [ ] I have read and understood SpaceVim's [CONTRIBUTING](https://github.com/SpaceVim/SpaceVim/blob/master/CONTRIBUTING.md) document.
- [ ] I have read and understood SpaceVim's [CODE_OF_CONDUCT](https://github.com/SpaceVim/SpaceVim/blob/master/CODE_OF_CONDUCT.md) document.
- [ ] I understand my PR may be closed if it becomes obvious I didn't actually perform all of these steps.

### Why this change is necessary and useful?

[Please explain **in detail** why the changes in this PR are needed.]

### CODE REVIEW

File: docs/_posts/2019-02-17-use-vim-as-a-ruby-ide.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+该模块会根据当前环境自动在多种补全引擎之间选择合适的，
+默认的补全引擎有：deoplete、neocomplete、ycm、asyncomplete 以及 neocomplcache。
+几种自动补全引擎当中，要数 deoplete 的体验效果最好。
+
+![perlcomplete](https://user-images.githubusercontent.com/13142418/52611209-54550500-2ebf-11e9-9b9f-f697a0db52a3.png)
+
+### 语法检查
+
+`checkers` 模块为 SpaceVim 提供了语法检查的功能，该模块默认已经载入。该模块默认使用 [neomake](https://github.com/neomake/neomake)
+这一异步语法检查工具。对于 Ruby 的支持，是通过异步调用 ruby 和 perlcritic 命令来完成的。
```
Comment (xylinkun) [inline docs/_posts/2019-02-17-use-vim-as-a-ruby-ide.md]: I guess all the words begin with `perl` are not correct.

**Comment by wsdjeg** [comment]
rubyrunner

**Comment by wsdjeg** [comment]
formatruby

**Comment by wsdjeg** [comment]
repl

**Comment by wsdjeg** [comment]
rubylint

**Comment by wsdjeg** [comment]
rubyrepl

**Comment by wsdjeg** [comment]
rubyide

**Comment by wsdjeg** [comment]
rubyomni

**Pull request merged**